src/client/index.ts

```diff
@@ -104,35 +95,57 @@ export class Client {
 		this.messaging = new Messaging();
 	}
 
+	formatCollectionID(collectionID: string) {
+		let formatedCollectionID = collectionID;
+
+		if (/[A-Z]+/g.test(collectionID) || /\s+/g.test(collectionID)) {
+			formatedCollectionID = collectionID.replace(/\s+/g, "-").toLowerCase();
```

Comment:
if its important to us to stay only low-case letters, digits and "-" then we have to replace/remove all special symbols too.
is this allowed for example? "df&`2?\n\t*}}"
so better stay only allowed symbols, not just lowercase them.
@micwallace , @nicktaras